go.sum

```diff
@@ -1,7 +1,5 @@
-cloud.google.com/go v0.26.0 h1:e0WKqKTd5BnrG8aKH3J3h+QvEIQtSUcf2n5UZ5ZgLtQ=
```

Comment:
Why is this file modified?

Reply:
Go 1.16 gets rid of those entries when you run `go mod tidy`. I don't know the reason why. The changes seem to be harmless given that the E2E tests ran successfully so I kept them on.